src/Twig/PossibleTwigMethodCallsProvider.php

```diff
@@ -71,15 +76,26 @@ public function provide(): array
     }
 
     /**
-     * @return SplFileInfo[]
+     * @return list<string>
      */
-    private function findTwigFileInfos(string $directory): array
+    private function findTwigFiles(string $directory): array
     {
-        $twigFinder = Finder::create()
-            ->files()
-            ->in($directory)
-            ->name('*.twig');
+        $it = new GlobIterator($directory . '/*.twig');
```

Comment:
Btw, does this work for nested files too? E.g.

```bash
directory/another-directory/*.twig?
```

Current test does not support this one, so I'm not sure:
https://github.com/TomasVotruba/unused-public/tree/main/tests/Rules/UnusedPublicClassMethodRule/templates

Reply:
no it doesnt. fixed with https://github.com/TomasVotruba/unused-public/pull/22